xc7/utils/prjxray_create_edges.py

```diff
@@ -450,9 +448,8 @@ def find_connector(wire_pkey, node_pkey):
 
     return find_connector
 
-def make_connection(conn, input_only_nodes, output_only_nodes,
-        find_wire, find_pip, find_connector,
-        tile_name, loc, tile_type, pip, switch_pkey):
+
+def make_connection(conn, input_only_nodes, output_only_nodes, loc, tile_pkey, src_wire_pkey, dst_wire_pkey, pip_pkey, switch_pkey, find_connector):
```

Comment:
Why did this get reverted?  The previous format was conforming to the max line length of 80.

Reply:
I didn't know that I should keep line lengths < 80. I will fix that.

Reply:
We just add back on auto-formatting with the target "make python_format", and the CI is checking formatting for green.

Reply:
I did the formatting.